lib/sequent/migrations/view_schema.rb

```diff
@@ -283,6 +282,27 @@ def migrate_offline
         raise e
       end
 
+      def migrate_dryrun(regex, group_exponent)
+        # Complete hack to override insert_ids
+        define_singleton_method(:insert_ids) do
+          ->(progress, done, ids) {}
+        end
+
+        persistor = Sequent.configuration.online_replay_persistor_class.new
```

Comment:
This depends on the `online_replay_persistor_class` begin used if this works or not. 
E.g. the `ActiveRecordPersistor` has no commit since that is commit when the transaction is commit.

https://github.com/zilverline/sequent/blob/29b40ffc06bdd4ca2595bad69a1256988778e3fd/lib/sequent/core/persistors/active_record_persistor.rb#L120-L122

Since this is configurable I think we should implement our own ReadonlyReplayPersistor that we set in the rake task in the sequent config to be used. Then there is no need paths this class.

Perhaps the same applies to introducting a subclass of `ViewSchema`